action-server/alembic/versions/2c2215d3e953_initial_db_setup.py

```diff
@@ -0,0 +1,124 @@
+"""Initial db setup.
+
+Revision ID: 2c2215d3e953
+Revises:
+Create Date: 2020-04-21 13:32:44.422928
+
+"""
+import structlog
+from sqlalchemy import Boolean, Column, Date, DateTime, String, Time, func
+from sqlalchemy.dialects.postgresql import ENUM, UUID
+from sqlalchemy.sql.schema import ForeignKey
+
+from alembic import op
+
+log = structlog.get_logger(__name__)
+
+revision = "2c2215d3e953"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+# create_type=False for ENUM: see
+# https://bitbucket.org/zzzeek/alembic/issues/53/create-table-dont-create-enum
+
+canadian_provinces = ENUM(
+    "AB",
+    "BC",
+    "MB",
+    "NB",
+    "NL",
+    "NS",
+    "NT",
+    "NU",
+    "ON",
+    "PE",
+    "QC",
+    "SK",
+    "YT",
+    name="canadian_province",
+    create_type=False,
+)
+
+languages = ENUM("en", "fr", name="language", create_type=False,)
+
+symptoms_severity = ENUM(
+    "none", "moderate", "severe", name="symptoms_severity", create_type=False,
```

Comment:
Those are not the same values as in https://docs.google.com/spreadsheets/d/1ZscIRkGI9UwKNsby7duw08Gp1LUaM4s_nKEXafXOZoM

is it normal?

Reply:
It took the value from here: https://docs.google.com/document/d/1MYCsg5zhvf95gBq7W1R79eR23qhtuIY2gSaQJUZBRkU

But you're right it should be the same.